docs/guides/admin/docs/configuration/playlists.md

```diff
@@ -0,0 +1,8 @@
+Playlists
+=========
+
+A list of videos, similar to the same concept in YouTube. This provides an n:m mapping, meaning that every playlist can
+contain multiple videos, and each video can be included in multiple playlists. (As opposed to the 1:n mapping of
+Opencast series, where each video is part of at most one series.)
+
+Playlists are currently only available through their rest endpoint and don't show up anywhere else.
```

Comment:
I think this could use a bit more information, e.g. the fact that the order of playlist entries are significant and it's not a "collection"/"set".